.gitattributes

```diff
@@ -0,0 +1,2 @@
+.github/workflows/sycl_precommit.yml merge=ours
+.github/CODEOWNERS merge=ours
```

Comment:
```suggestion
.github/CODEOWNERS merge=ours
```
Thanks @jsji , good to know there is a way to automatically merge with ours, but I would only do it for CODEOWNERS for now.

Reply:
OK. Updated.